boards/arm/efm32/efm32-g8xx-stk/scripts/Make.defs

```diff
@@ -51,8 +51,8 @@ CXXPICFLAGS = $(ARCHPICFLAGS) $(CXXFLAGS)
 CPPFLAGS := $(ARCHINCLUDES) $(ARCHDEFINES) $(EXTRAFLAGS)
 AFLAGS := $(CFLAGS) -D__ASSEMBLY__
 
-NXFLATLDFLAGS1 = -r -d -warn-common
-NXFLATLDFLAGS2 = $(NXFLATLDFLAGS1) -T$(TOPDIR)/binfmt/libnxflat/gnu-nxflat-pcrel.ld -no-check-sections
+NXFLATLDFLAGS1 = -r -Wl,-d -Wl,-warn-common
+NXFLATLDFLAGS2 = $(NXFLATLDFLAGS1) -T$(TOPDIR)/binfmt/libnxflat/gnu-nxflat-pcrel.ld -Wl,-no-check-sections
```

Comment:
I don't know if this should be addressed by this PR, but it seems the correct flag is `--no-check-sections`:
https://sourceware.org/binutils/docs/ld/Options.html#:~:text=--no-check-sections

Reply:
`--warn-common` is also the correct, with two dashes.
Let's address everything later.

Reply:
I don't see difference

Reply:
https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_node/ld_3.html#:~:text=for%20options%20whose%20names%20are%20multiple%20letters%2C%20either%20one%20dash%20or%20two%20can%20precede%20the%20option%20name

Yeah, LD accepts both ways.